src/main/java/com/sourcegraph/cody/SubscriptionPanel.kt

```diff
@@ -4,11 +4,52 @@ import com.intellij.ide.BrowserUtil
 import com.intellij.ide.ui.laf.darcula.ui.DarculaButtonUI
 import com.intellij.ui.dsl.builder.panel
 import com.sourcegraph.common.CodyBundle
+import com.sourcegraph.common.UpgradeToCodyProNotification
 import com.sourcegraph.config.ConfigUtil
+import com.sourcegraph.config.ThemeUtil
 
 fun createSubscriptionTab(isCurrentUserPro: Boolean) = panel {
+  val chatLimitError = UpgradeToCodyProNotification.chatRateLimitError.get()
+  val autocompleteLimitError = UpgradeToCodyProNotification.autocompleteRateLimitError.get()
+  val chatLimitLabelValue =
+      "You used all ${chatLimitError?.limit} chat messages for this month. Upgrade to Cody Pro to get unlimited chats."
+  val autocompleteLimitLabelValue =
+      "You used all ${autocompleteLimitError?.limit} autocompletions for this month. Upgrade to Cody Pro to get unlimited autocompletions."
+  val chatAndAutocompleteLimitLabelValue =
+      "You used all you autocompletions and chats for this month. Upgrade to Cody Pro to get unlimited interactions."
+  if (!isCurrentUserPro) {
+    row {
+      text(
+          "<html>" +
+              "<table width=\"100%\">" +
+              "<tr>" +
+              "<td width=\"10%\"><span style=\"font-size:20px;\">⚡</span></td>" +
+              "<td width=\"90%\"><p>${
+                if (chatLimitError != null && autocompleteLimitError != null) {
+                  chatAndAutocompleteLimitLabelValue
+                } else {
+                  if (chatLimitError != null) {
+                    chatLimitLabelValue
+                  } else {
+                    autocompleteLimitLabelValue
+                  }
+                }
+                }</p></td>" +
+              "</tr>" +
+              "</table>" +
+              "</html>")
+    }
+    row {
+      if (ThemeUtil.isDarkTheme()) {
```

Comment:
What is you switch theme when this tab is visible? 
Will it switch colours properly?

Reply:
Valid issue, i tested it before and it doesnt change the colour properly but i am hesitating to do the rerendering of the whole tab...

Reply:
I checked and it doesnt work automatically